packages/common/utils/algoliaHelpers.js

```diff
@@ -47,13 +60,13 @@ export async function getClient(appId, apiKey, server, userId, noSignature) {
     if (!isLocalAppId(appId) && apiKey && noSignature !== true) {
         const signature = await getSignature(appId);
         if (signature) {
-            headers['X-Algolia-Signature'] = signature;
+            headers["X-Algolia-Signature"] = signature;
         }
     }
 
     const logger = createConsoleLogger(1);
     const cache = createNullCache();
-    const baseClient = algoliasearch(appId, apiKey || ' ', {
+    const baseClient = algoliasearch(appId, apiKey || " ", {
```

Comment:
Do you think it would be possible to modify the algolia-user-agent passed here to send an agent identifier which would contain the fact that this was done from Metaparams and a hash of the user email?

This would allow tracing in all of our other logging facilities (like in logger.algolia.com)